Plugin/Quote/Model/BillingAddressManagement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Yotpo\SmsBump\Plugin\Quote\Model;
+
+use Magento\Quote\Model\BillingAddressManagement as QuoteBillingAddressManagement;
+use \Magento\Quote\Api\Data\AddressInterface;
```

Comment:
does this work with `\Magento\`? should it just be `Magento\`?

Reply:
It's the same.